storage/zfs/zpool-importer.yaml

```diff
@@ -0,0 +1,50 @@
+name: zpool-importer
+depends:
+  - service: udevd
+  - service: cri
+  - path: /dev/zfs
+container:
+  entrypoint: /usr/local/sbin/zpool-importer
+  mounts:
+    - source: /dev
+      destination: /dev
+      type: bind
+      options:
+        - rshared
+        - rbind
+        - rw
+    # ld-musl-x86_64.so.1
```

Comment:
for my understanding, if the extension anyway execs in pid 1 mount namespace, why do we need all the mounts?

Reply:
ahh good point, I'll drop those and test

Reply:
ahh it still needs those:

```bash
10.5.0.3: 2023/08/21 12:13:28 zpool-importer: reexec failed: gons/reexec: ReexecAction.Run: child failed with stderr message "err: exit status 1, /dev/zfs and /proc/self/mounts are required.\nTry running 'udevadm trigger' and 'mount -t proc proc /proc' as root.\n"
```

I think i starts from the parent and targets the mount of pid 1

Reply:
hmmm... I'm not sure I understand this...

Reply:
okay `/dev` is not needed, the other mounts are shared libraries, from the looks of it it seems the shared libs are needed in the parent mount namespace